<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="fanSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>101</width>
      <height>285</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical {
    border-radius: 1px;
    width: 2px;
    margin: 24px;
    background-color: rgb(52, 59, 72);
}

QSlider::handle:vertical {
    background-color: rgb(255, 255, 255);
    border: 4px solid black;
    height: 38px;
    width: 20px;
    margin: -24px -24px;
    border-radius: 19px;

}
QSlider::sub-page:vertical {
    width: 2px;
    margin: 0px 24px; 
    background-color: rgb(52, 59, 72);
}
QSlider::add-page:vertical { 
	background-color:  rgb(255, 255, 255); 
	width: 2px;
	margin: 0px 24px; 
} </string>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>13</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fan.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fanLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 28pt &quot;Modern&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="both_i">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>both.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="defrost_i">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>defrost.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="feetDef_i">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>defrostFeet.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="feet_i">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>feet.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="head_i">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>head.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="heatSeater_i">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>otherSeat.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="both_L">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 174, 233);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="feet_L">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="head_L">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="feetDef_L">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="defrost_L">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heatSeater_L">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="both">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="feet">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="head">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="feetDef">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="defrost">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="heatSeater">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="tempSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>961</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border-radius: 1px;
    height: 2px;
    margin: 24px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(30, 0, 255), stop: 1 rgb(255, 0, 0));
}

QSlider::handle:horizontal {
    background-color: rgb(255, 255, 255);
    border: 4px solid black;
    height: 20px;
    width: 38px;
    margin: -24px -24px;
    border-radius: 19px;
}
/*
QSlider::sub-page:horizontal {
    height: 2px;
    margin: 20px 0px; 
    background-color: rgb(255, 255, 255);

}
QSlider::add-page:horizontal {
    height: 2px;
    margin: 20px 0px; 
    background-color: rgb(52, 59, 72);
}
*/</string>
    </property>
    <property name="minimum">
     <number>60</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="insideAir_i">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>insideAir.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outsideAir_i_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>hot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outsideAir_i_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cold.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outsideAir_i">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>outsideAir.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="heatSeater_L_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>295</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heatSeater_i_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>heatSetaer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="heatSeater_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outsideTemp">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>OUTSIDE: 70°</string>
    </property>
   </widget>
   <widget class="QLabel" name="AC_i">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>A/C</string>
    </property>
   </widget>
   <widget class="QLabel" name="insideAir_L">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 174, 233);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AC_L">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="outsideAir_L">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="insideAir">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="outsideAir">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dayLight_i">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>headlights.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dayLight_L">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>61</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 59, 72);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dayLight">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>INSIDE:    70°</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLabel">
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>37</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>15°</string>
    </property>
   </widget>
   <widget class="QLabel" name="rollLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>37</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>15°</string>
    </property>
   </widget>
   <widget class="QSlider" name="rollSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>713</x>
      <y>63</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border-radius: 1px;
    height: 2px;
    background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.5, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(30, 0, 255, 255));
}

QSlider::handle:horizontal {
    background-color: rgb(255, 255, 255);
    border: 4px solid black;
    height: 10px;
    width: 10px;
    margin: -10px -10px;
    border-radius: 19px;
}
/*
QSlider::sub-page:horizontal {
    height: 2px;
    margin: 20px 0px; 
    background-color: rgb(255, 255, 255);

}
QSlider::add-page:horizontal {
    height: 2px;
    margin: 20px 0px; 
    background-color: rgb(52, 59, 72);
}
*/</string>
    </property>
    <property name="minimum">
     <number>-25</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="pitchSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical {
    border-radius: 1px;
    width: 2px;
    background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:0.5, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(20, 0, 255, 255));
}

QSlider::handle:vertical {
    background-color: rgb(255, 255, 255);
    border: 4px solid black;
    height: 10px;
    width: 10px;
    margin: -10px -10px;
    border-radius: 19px;

}
</string>
    </property>
    <property name="minimum">
     <number>-25</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <zorder>fanSlider</zorder>
   <zorder>label</zorder>
   <zorder>fanLabel</zorder>
   <zorder>both_i</zorder>
   <zorder>defrost_i</zorder>
   <zorder>feetDef_i</zorder>
   <zorder>feet_i</zorder>
   <zorder>head_i</zorder>
   <zorder>heatSeater_i</zorder>
   <zorder>both_L</zorder>
   <zorder>feet_L</zorder>
   <zorder>head_L</zorder>
   <zorder>feetDef_L</zorder>
   <zorder>defrost_L</zorder>
   <zorder>heatSeater_L</zorder>
   <zorder>both</zorder>
   <zorder>feet</zorder>
   <zorder>head</zorder>
   <zorder>feetDef</zorder>
   <zorder>defrost</zorder>
   <zorder>heatSeater</zorder>
   <zorder>tempSlider</zorder>
   <zorder>insideAir_i</zorder>
   <zorder>outsideAir_i_2</zorder>
   <zorder>outsideAir_i_3</zorder>
   <zorder>outsideAir_i</zorder>
   <zorder>heatSeater_L_2</zorder>
   <zorder>heatSeater_i_2</zorder>
   <zorder>heatSeater_2</zorder>
   <zorder>outsideTemp</zorder>
   <zorder>AC_i</zorder>
   <zorder>insideAir_L</zorder>
   <zorder>AC_L</zorder>
   <zorder>outsideAir_L</zorder>
   <zorder>insideAir</zorder>
   <zorder>AC</zorder>
   <zorder>outsideAir</zorder>
   <zorder>dayLight_i</zorder>
   <zorder>dayLight_L</zorder>
   <zorder>dayLight</zorder>
   <zorder>label_4</zorder>
   <zorder>rollLabel</zorder>
   <zorder>pitchLabel</zorder>
   <zorder>rollSlider</zorder>
   <zorder>pitchSlider</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fanSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fanLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
